### ISSUE TITLE
500 Server error

### ISSUE THREAD

**Issue by venticello** [Cosmologist/RussianPost#1]
Тестовые данные не менял (точь в точь как в описании), при исполнении кода ошибка сервера Почты России.

**Comment by Cosmologist** [comment]
Да, такое может быть. За несколько месяцев использования библиотеки, API у них менялось несколько раз - под конец, я уже устал постоянно реверс-инжинирить их изменения и забил.
Попробую посмотреть на выходных - если быстро поправить не получится, то удалю репозиторий.

**Comment by venticello** [comment]
Не надо удалять, можно просто написать в readme, что адрес доступа поменялся и сейчас возможно скрипт не работает.
Возможно кому то пригодятся наработки. Спасибо.
